plugins/ui/src/js/src/UITable.tsx

```diff
@@ -25,27 +25,22 @@ function UITable({
   const dh = useApi();
   const [model, setModel] = useState<IrisGridModel>();
   const [columns, setColumns] = useState<Table['columns']>();
+  const [utils] = useState(new IrisGridUtils(dh));
```

Comment:
Should `useMemo` instead - doing a `useState` will not update it if `dh` changes.
```suggestion
  const utils = useMemo(() => new IrisGridUtils(dh), [dh]);
```